### PR TITLE
Add new feature for ems

### PR DESCRIPTION
**Pull request by shivamchouhan123**
description:   Add a new feature for ems

### CODE REVIEW

**Review by salvimohan — COMMENTED** [review]
why total hours and total minutes calculate on save

File: db/migrate/20230907122843_create_logged_hours.rb
Comment (salvimohan) [inline db/migrate/20230907122843_create_logged_hours.rb:1]: This table require task id time column which save hours and time

File: app/models/employee.rb [hunk 1]
```diff
@@ -11,6 +11,10 @@ class Employee < ApplicationRecord
   has_many :performances, dependent: :destroy
   has_many :salaries, dependent: :destroy
   has_many :attendences, dependent: :destroy
+  has_many :logged_hours, dependent: :destroy
+  has_many :tasks, dependent: :destroy
+  has_many :notes, dependent: :destroy
+  belongs_to  :project
```
Comment (salvimohan) [inline app/models/employee.rb:17]: Employee also have many projects through task

**Comment by salvimohan** [comment]
why we are not using TaskJuggler gem for task management

File: db/migrate/20230907122843_create_logged_hours.rb
Comment (shivamchouhan123) [reply db/migrate/20230907122843_create_logged_hours.rb:1]: Task id already exists and as far as Total_Hour and Total_Minute are concerned, I have added the attribute of Total_Time by changing it

File: app/models/employee.rb (hunk 1 above)
Comment (shivamchouhan123) [reply app/models/employee.rb:17]: resolve these issue

**Comment by shivamchouhan123** [comment]
> why total hours and total minutes calculate on save

**Comment by shivamchouhan123** [comment]
> why we are not using TaskJuggler gem for task management

I didn't know about this but I'll add it.

**Pull request merged**